File: 16-fizzbuzz/fizzbuzz.exs
# Elixir program to print FizzBuzz
# Created by M.V.Harish Kumar on 15-01-2024

{n, _} = IO.gets("Enter no. of terms: ") |> Integer.parse
for i <- 1..n do
  cond do
    rem(i, 15) == 0 -> "FizzBuzz"
    rem(i, 3) == 0  -> "Fizz"
    rem(i, 5) == 0  -> "Buzz"
    true -> i
  end |> IO.puts
end

File: 16-fizzbuzz/fizzbuzz.f90
! Fortran program to print fizzbuzz
! Written by M.V.Harish Kumar on 11-05-2024
program fizzbuzz
  implicit none
  integer :: n, i

  print '(a$)', "Enter no. of terms: "
  read(*,*) n

  do i = 1, n
     if (mod(i,15)==0) then
        print '(a)', "FizzBuzz"
     else if (mod(i,3)==0) then
        print '(a)', "Fizz"
     else if (mod(i,5)==0) then
        print '(a)', "Buzz"
     else
        print '(i0)', i
     end if
   
  end do
end program fizzbuzz
